src/components/InterestForm/styles.tsx

```diff
@@ -28,34 +28,32 @@ export const FormTextArea = styled.textarea`
   border-radius: 0.313rem; // 5px
   border: 2px solid ${COLORS.greyLight};
   margin-bottom: 0.6rem;
-  padding: 0.5rem;
+  padding: 0.625rem 1.25rem; // 10px 20px 10px 20px;
   &:focus {
     border-color: ${COLORS.blueMid};
     outline: none;
   }
-`;
+  &::placeholder {
+    color: ${COLORS.placeholderText};
+  }
 
-export const FormInput = styled.input`
-  ${openSans.style}
-  font-size: 1rem;
-  font-weight: 400;
-  border-radius: 0.313rem; // 5px
-  border: 2px solid ${COLORS.greyLight};
-  padding: 0.5rem;
-  margin-bottom: 0.6rem;
-  &:focus {
-    border-color: ${COLORS.blueMid};
-    outline: none;
+  &:not(:placeholder-shown) {
+    border-color: ${COLORS.greyMid};
   }
 `;
 
 export const RadioGroup = styled.div`
   display: flex;
   flex-direction: row;
-  margin-bottom: 0.6rem;
+  align-self: center;
```

Comment:
think this might be causing the set of radio buttons to be shifted to the right horizontally? 

<img width="530" alt="Screenshot 2023-11-23 at 3 35 38 PM" src="https://github.com/calblueprint/immigration-justice-project/assets/47801900/ede72bd0-8fe3-4152-888f-0c0bd6369fb1">